client/tinychain/app.py

```diff
@@ -78,6 +93,10 @@ def __ns__(self, _context, _name_hint):
     def __ref__(self, name):
         return ModelRef(self, name)
 
+    @classmethod
+    def key(cls):
```

Comment:
It might make more sense to put this in the same module as `create_schema` since that's the only place it's useful

Reply:
The reason i have it in the Model class is that it's only really relevant for the Model.